source/library/Witch/Instances.hs

```diff
@@ -1272,6 +1275,21 @@ instance From.From Time.NominalDiffTime Time.CalendarDiffTime where
 instance From.From Time.ZonedTime Time.UTCTime where
   from = Time.zonedTimeToUTC
 
+-- Tagged
+
+-- | Uses @coerce@.
+instance From.From a (Tagged.Tagged s a)
+
+-- | Uses @coerce@.
+instance From.From (Tagged.Tagged s a) a
+
+-- | Uses @coerce@.
+instance From.From (Tagged.Tagged s a) (Tagged.Tagged t a)
+
+-- | Uses 'From.from' on the tagged value.
+instance From.From a b => From.From (Tagged.Tagged s a) (Tagged.Tagged s b) where
+  from = fmap From.from
```

Comment:
I'm not sure that I should provide this instance. I have so far shied away from writing instances that distribute over types, like `From a b => From (Maybe a) (Maybe b)`. See #37 for a discussion of that.